.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -6,20 +6,4 @@ labels: ''
 assignees: ''
 ---
 
-**Is your feature request related to a problem? Please describe**
-
-A clear and concise description of the problem you're trying to solve.
-
-**Describe the desired solution**
-
-A clear and concise description of a new feature and its application. For
-example, "It would be great if Cantera could..."
-
-**Describe alternatives you have considered**
-
-A clear and concise description of any alternative solutions or features you
-have considered.
-
-**Additional context**
-
-Add any other context about the feature request here.
+Feature requests have been moved to [Cantera/enhancements](https://github.com/Cantera/enhancements) and should be opened there. Thank you for your suggestions!
```

Comment:
It would be good to link this to https://github.com/Cantera/enhancements/issues/new so the link goes straight to creating a new issue

Reply:
You're welcome. I added this to the existing link to keep things simple (unfortunately links won't get rendered unless in preview mode)

Reply:
You can link to https://github.com/Cantera/enhancements/issues/new/choose to get the template chooser

Reply:
Or, probably better since this is replacing the Feature Request template here, is to link directly to the feature request template: https://github.com/Cantera/enhancements/issues/new?assignees=&labels=feature-request&template=feature_request.md&title=

Reply:
Those are possibilities. The link is, however, not clickable (unless in preview mode), so I am still wondering whether the shortest option isn’t the most readable. I don’t have strong preferences though.

Reply:
Oh, I see what you're saying. Then probably a raw link to the chooser would be the best bet, that's not that complicated.

Reply:
... thanks the suggestions. I went with the chooser, but retained the markdown (in case anyone hits preview to click, rather than cut&pasting the link)